about.md

```diff
@@ -5,31 +5,33 @@ description: The A11Y Project aims to make accessibility easier
+There are a number of ways a numeronym can be created. In the case of translating "accessibility" to "A11Y", the first and last letters of "accessibility" are preserved, replacing the remaining letters between with their sum total of 11.
 
-One of the motivations for a numeronym is the limited space available on social media platforms for posting content. For example, on Twitter - where the character limit was 140 (until November 2017) - by replacing the hashtag [#accessibility](https://twitter.com/hashtag/accessibility) with [#a11y](https://twitter.com/hashtag/a11y), the author now has another 9 characters available to them. Easily enough for another one or two important words on the subject of accessibility.
+One of the motivations for a numeronym is the limited space available on social media platforms for posting content. For example, on Twitter—where the character limit was 140 until November 2017—replacing the hashtag [#accessibility](https://twitter.com/hashtag/accessibility) with [#a11y](https://twitter.com/hashtag/a11y) freed up 9 extra characters for the author to use.
 
-Another motivation is discoverability. Searching for this subject with “accessibility”, you will find many irrelevant resources using that word in various other senses, and you will also miss resources in languages other than English. To the extent that specialists use “a11y” as an international technical term, you can find more relevant resources, and with greater precision.
+Another motivation is discoverability. You can find more relevant resources, and with greater precision. Using the term “accessibility” to search, you will find many irrelevant resources using that word in various other contexts. You may also miss resources in languages other than English. Many accessibility specialists from around the world use “a11y” as an international technical term when speaking about their work.
 
 
 ### About this site
 
-This site is built with [Jekyll](https://github.com/mojombo/jekyll), collaboratively by developers (see the footer) on [Github](https://github.com/a11yproject/a11yproject.com/). In its current state, The Accessibility Project may not be a perfect example of an accessible website&hellip; yet. We're committed, through the power of Open Source, to incrementally improve our own accessibility on the site going forward.
+This site is built collaboratively by developers on [GitHub](https://github.com/a11yproject/a11yproject.com/) using [Jekyll](https://github.com/mojombo/jekyll).
+
+Through the power of Open Source we're committed to incrementally improving our own site's accessibility. If you are knowledgeable about accessibility and want to help, please consider [contributing](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md). We would love people to help write articles, edit inaccuracies, and provide high-level information.
```

Comment:
"We would love people to help...," sounds a little like people don't already do this. Maybe it could be more active, like "We always welcome people to help...," or "Everyone is welcome to help..."?

Reply:
Good call. I'll update this now and merge.